### DISCUSSION TITLE
Impact of Precoding on Total CIR and Path Coefficient Calculation in Multi-Antenna Scenarios

### DISCUSSION THREAD

**Discussion by alialsahlany** [NVlabs/sionna#586] [General]
How does the use of precoding affect the total CIR, and how can the value for each path be calculated in multi-antenna scenarios?

**Comment by faycalaa** [comment]
Hi @alialsahlany ,

The channel models and ray tracer compute a path coefficient and delay for each pair of transmit and receive antennas. This process is independent of the precoding vector.

**Reply by alialsahlany** [reply]
Hi @faycalaa , many tahnks for your quick reply.

How can I calculate the CIR in a 2x2 MIMO system at both the transmitter and the receiver?

**Reply by faycalaa** [reply]
I recommend checking out the [tutorials](https://nvlabs.github.io/sionna/tutorials.html), such as [this one](https://nvlabs.github.io/sionna/examples/Realistic_Multiuser_MIMO_Simulations.html) on MIMO simulations.

**Reply by alialsahlany** [reply]
> I recommend checking out the [tutorials](https://nvlabs.github.io/sionna/tutorials.html), such as [this one](https://nvlabs.github.io/sionna/examples/Realistic_Multiuser_MIMO_Simulations.html) on MIMO simulations.

I have read this post, but it is not helpful. I need to calculate the CIR for multiple paths outgoing to multi antennas.
